### DISCUSSION TITLE
网站正式启用 - 晦涩弗里曼的博客

### DISCUSSION THREAD

**Discussion by giscus** [obscurefreeman/blog#6] [Giscus]
# 网站正式启用 - 晦涩弗里曼的博客

不管会发生什么，网站现在已经被正式启用了，现在是2023年10月26日。

https://obscurefreeman.github.io/blog/%E7%BD%91%E7%AB%99%E4%BA%8B%E4%BB%B6/%E7%BD%91%E7%AB%99%E6%AD%A3%E5%BC%8F%E5%90%AF%E7%94%A8/

**Comment by obscurefreeman** [comment]
欢迎来到我的网站，都来发评论吧！
